README.md

```diff
@@ -9,6 +9,9 @@ component repositories because they operate above a source-code level, are
 optional, and they can work with multiple PDC services above any given PDC
 service's level.
 
+This README document describes components of a deployment pipeline. For specific
+steps to deploy using the existing main deployment pipeline, see [deploying.md].
```

Comment:
Did you test that all the Markdown renders as expected?  I don't think that "[deploying.md]" does what you think it does.  (You can render Markdown->HTML locally to test -- [this function](https://viewvc.red-bean.com/kfogel/trunk/.emacs?revision=6560&view=markup#l6828) in my `.emacs` is how I do it from Emacs, but you could just swipe the `pandoc -f gfm ...` command from there and use it however you want, of course.)

Reply:
Also, you can just say "This README" instead of "This README document" :-).  (It's like saying "the `foo` variable has the value 3" instead of just "`foo` has the value 3".)

Reply:
> Did you test that all the Markdown renders as expected? I don't think that "[deploying.md]" does what you think it does.

D'oh, nope.

> (You can render Markdown->HTML locally to test -- [this function](https://viewvc.red-bean.com/kfogel/trunk/.emacs?revision=6560&view=markup#l6828) in my `.emacs` is how I do it from Emacs, but you could just swipe the `pandoc -f gfm ...` command from there and use it however you want, of course.)

This is now in my `init.el`. After installing pandoc and markdown it just works. I also found a system-level package of ELPA's markdown mode and that helps too.

Reply:
Thanks to your Emacs function, it was easy to test the links! Thank you!

Reply:
Ah, great! You're welcome.  I use that function all the time.